docs/cunumeric/source/api/ndarray.rst

```diff
@@ -303,4 +302,4 @@ String representations:
 .. .. autosummary::
 ..    :toctree: generated/
 
-..    ndarray.__class_getitem__
\ No newline at end of file
+..    ndarray.__class_getitem__
```

Comment:
This shows up because a newline was added at the end of the file, where none existed before. I think we should always have a newline at the end of files, so this change is welcome in my opinion.